backend/app/api/content_machine.py

```
@@ -0,0 +1,314 @@
+            
+            if not bookmarks_path.exists():
+                raise HTTPException(status_code=404, detail="Bookmarks directory not found")
+            
+            result = await absorb_from_bookmarks(style_db, request.writer_name, bookmarks_path)
+            
+            if result:
+                return AbsorbStyleResponse(
+                    writer_name=request.writer_name,
+                    success=True,
+                    sample_count=result.get("sample_count", 0),
+                    message=f"Successfully absorbed style from bookmarks",
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)